includes/AppLeave.php

```diff
@@ -145,6 +149,7 @@
 	//echo "Updating application for $user.  ";
 	
 	$userDB = str_replace('.','$',$user);
+	echo $userDB;
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->